packages/cli/src/WaitTracker.ts

```diff
@@ -186,7 +186,8 @@ export class WaitTracker {
 			if (!fullExecutionData.workflowData.id) {
 				throw new Error('Only saved workflows can be resumed.');
 			}
-			const user = await getWorkflowOwner(fullExecutionData.workflowData.id);
+			const workflowId = fullExecutionData.workflowData.id;
+			const user = await Container.get(OwnershipService).getWorkflowOwnerCached(workflowId);
```

Comment:
`OwnershipService` could be injected here since `WaitTracker` is also a service.